firebase/database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    ".read": "now < 1617832800000",  // 2021-4-8
+    ".write": "now < 1617832800000",  // 2021-4-8
+  }
+}
```

Comment:
I don't know if firebase is ok with that, but comments are NOT part of json. There's no such thing as comments in json. That's why people sometimes use yaml

Reply:
Mhhmmm moreover, not sure what's the purpose of this. If you look on the firebase console, I disabled the rules (aka you can always read/write freely for now). Here, you say that a person may only read/write using the emulator if they have a time mahcine right?

Reply:
I did not generate those files, firebase-tools did. I don't think the rules are useful so we could delete them